src/plugins/tagfilter/tagfilter.js

```diff
@@ -214,6 +222,21 @@ const componentName = "wb-tagfilter",
 		refineFilters( instance );
 		matchItemsToFilters( instance );
 		updateDOMItems( instance );
+
+		// Show no result message if on Firefox -- Remove once Firefox supports ":has()"
+		if ( navigator.userAgent.toLowerCase().indexOf( "firefox" ) > -1 ) {
```

Comment:
It will be preferable to do a [feature detection](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection) instead of being browser specific. Being being browser specific could let to unexpected behaviour when the browser would start to support the feature, especially when we know that some site take a lot of time before to update their WET-BOEW implementation to the latest version.

Can you check if it would be possible to use a polyfill?
I didn't tested but like: https://github.com/jplhomer/polyfill-css-has/blob/master/src/polyfill-css-has.ts
Can we leverage somehow the [@supports CSS at-rule](https://developer.mozilla.org/en-US/docs/Web/CSS/@supports) for this? Like if supported we style a little something that we can detect in JS to use an alternate implementation?